### ISSUE TITLE
Error in Mediso Dynamic PET Conversion

### ISSUE THREAD

**Issue by QinQikai** [rordenlab/dcm2niix#853]
Hi,

I met a problem when I use dcm2niix (v1.0.20240202 and v1.0.20240812) to convert Mediso Dynamic PET DICOM to NIfTI. The output images are in the wrong order. It seems dcm2niix cannot recognize “(0020,9057) InStackPositionNumber” and “(0020,9128) TemporalPositionIndex". 

In addition, each DICOM frame has its own “(0028,1053) RescaleSlope”. When I use dcm2niix (v1.0.20220720), it seems only use the first slope value. I am wandering if it was fixed in the latest version.

Thank you!

**Comment by neurolabusc** [comment]
dcm2niix should be able to use enhanced DICOM ordering. It does handle variable rescale slopes, but this feature is specific to Philips as I have not seen it in use by other manufacturers. If you want me to investigate changing the behavior of the software, you will need to share a concrete example with my [institutional email](https://sc.edu/study/colleges_schools/artsandsciences/psychology/our_people/directory/rorden_chris.php). Alternatively, create a pull request that handles these images.

**Comment by neurolabusc** [comment]
@QinQikai thank you for the sample images. I have added experimental support for the development branch to dcm2niix. Please validate that this works as expected.

```bash
git clone --branch development https://github.com/rordenlab/dcm2niix.gi...
cd dcm2niix/src
make
./dcm2niix /path/to/dicoms
...
Saving as 32-bit float (slope, intercept or bits allocated varies).
```

I do think you will need to work with the manufacturer to accurately populate the DICOM public tags to allow better [BIDS format JSON sidecars](https://github.com/rordenlab/dcm2niix/tree/development/BIDS#modality-positron-emission-tomography). For example "Bq/ml" belongs in the [Unit (0054,1001)](https://dicom.innolitics.com/ciods/positron-emission-tomography-image/pet-series/00541001) tag not the [Code Value (0008,0100)](https://dicom.innolitics.com/ciods/rt-dose/rt-series/00400260/00080100) tag. dcm2niix assumes that DICOM images are truthful and follow the specification. 

I do not have access to this hardware, and the provided sample datasets were overwhelmingly large, so please validate that the software works as expected.

**Comment by QinQikai** [comment]
@neurolabusc Thank you for adding the experimental support to the development branch of dcm2niix. I’ve cloned the repository and tested it with our DICOM datasets to validate that everything works as expected. I also invited other labs who encountered the same problem to test this version. 

By the way, we've contacted the manufacturer to address the issue. 
 
Thanks again for your support!

Just a reminder to others: 
Use `cd dcm2niix/console` instead of `cd dcm2niix/src`
